libminifi/src/core/extension/ExtensionManager.cpp

```diff
@@ -31,9 +31,13 @@ namespace org::apache::nifi::minifi::core::extension {
 
 const std::shared_ptr<logging::Logger> ExtensionManager::logger_ = logging::LoggerFactory<ExtensionManager>::getLogger();
 
-ExtensionManager::ExtensionManager() {
-  modules_.push_back(std::make_unique<Executable>());
-  active_module_ = modules_[0].get();
+ExtensionManager::ExtensionManager()
+    : modules_([] {
+        std::vector<std::unique_ptr<Module>> modules;
+        modules.push_back(std::make_unique<Executable>());
+        return modules;
+      }()),
```

Comment:
```suggestion
    : modules_{std::make_unique<Executable>()},
```

Reply:
Shoot, apparently the elements of an initializer_list are always passed by const reference, so it always attempts to copy...
https://stackoverflow.com/questions/8468774/can-i-list-initialize-a-vector-of-move-only-type

Reply:
Thanks for the reference!